codeHF/PlotEfficiency.C

```diff
@@ -51,49 +51,88 @@ Int_t PlotEfficiency(TString pathFile = "AnalysisResults.root", TString particle
 
     TString outputDir = Form("hf-task-%s", particle.Data()); // analysis output directory with histograms
 
+    TString nameHistRec;
+    TString nameHistgen;
+
     // inclusive candidates
-    TString nameHistRec = outputDir + "/hPtRecSig"; // reconstruction level pT of matched candidates
-    //nameHistRec = outputDir + "/hPtGenSig"; // generator level pT of matched candidates (no pT smearing)
-    TString nameHistgen = outputDir + "/hPtGen"; // generator level pT of generated particles
+    if (particles == " lc ") {
+      nameHistRec = "hf-task-lc/MC/reconstructed/signal/hPtRecSig"; // reconstruction level pT of matched candidates
+      nameHistgen = "hf-task-lc/MC/generated/signal/hPtGen";        // generator level pT of generated particles
```

Comment:
Why don't you use `outputDir` for Lc as well?

Reply:
I did not use the outputDir for the Lc because I needed to separate anyway between the different folders for generated and reconstructed. But since you suggested it I replaced the common part of the string with the outputDir.